packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -1409,11 +1409,6 @@ describe('<Autocomplete />', () => {
       }).toWarnDev([
         'returns duplicated headers',
         !strictModeDoubleLoggingSupressed && 'returns duplicated headers',
-        // React 18 Strict Effects run mount effects twice which lead to a cascading update
-        React.version.startsWith('18') && 'returns duplicated headers',
-        React.version.startsWith('18') &&
-          !strictModeDoubleLoggingSupressed &&
-          'returns duplicated headers',
```

Comment:
Is this change somehow related to useId?

Reply:
Yes! We avoid a cascading update so one commit less and therefore less error message in this test.